test_data/accessibility/stops.txt

```diff
@@ -0,0 +1,17 @@
+stop_id,stop_name,stop_desc,stop_lat,stop_lon,zone_id,stop_url,location_type,parent_station,wheelchair_boarding
+station_0,"station_0",,48.796058,2.449386,,,1,,
```

Comment:
Plenty of cases covering parent and child station.

Wheelchair information is provided for:

station_0 -> ❌
station_1 -> ❌
station_2 -> ✅
station_3 -> ✅
child_0 -> ❌
child_1 -> ❌
child_2 -> ✅ (inherit)
child_3 -> ✅
child_4 -> ✅
child_5 -> ✅ (inherit)
child_6 -> ❌
child_7 -> ❌
child_8 -> ✅ (inherit)
child_9 -> ✅
child_10 -> ✅
child_11 -> ✅ (inherit)